.travis.yml

```diff
@@ -7,7 +7,6 @@ git:
   submodules: false
 
 python:
-  - 2.7
```

Comment:
why?  aren't we testing here against released version of datalad, and testing datalad master against this in its own .travis.yml?

Reply:
Apparently not. Those builds failed with "datalad requires python >= 3.5"

Reply:
I believe since the beginning this has tested against a datalad installed from `git+https://github.com/datalad/datalad.git`.

Reply:
Double-checked: `requirements.txt` lets us install master.

Reply:
doh, indeed.  but I think that is a bug since none other extension (besides revolution) does it AFAIK, and it is against on what we have agreed in the past.  I will adjust #110 for that